packages/vscode-extension/src/panels/LaunchConfigController.ts

```diff
@@ -25,7 +25,9 @@ export class LaunchConfigController implements Disposable, LaunchConfig {
 
       const configurations = launchConfiguration.get<Array<Record<string, any>>>("configurations")!;
 
-      const RNIDEConfiguration = configurations.find(({ type }) => type === "react-native-ide");
+      const RNIDEConfiguration = configurations.find(
+        ({ type }) => type === "react-native-ide" || type === "radon-ide" // for compatibility we want to support old configuration type name
```

Comment:
I don't intent to keep them both actually. The plan is that in the follow up version we will be displaying messages if someone is using the old config asking to rename it. Would prefer to avoid adding complexity to handle a usecase that seem to be very unlikely